resolwe_bio/docker_images/rnaseq/packages-manual.txt

```diff
@@ -9,6 +9,7 @@ gotea
 genome-tools
 igvtools
 jbrowse
+makedb
```

Comment:
Rename this file to `walt-makedb` or something similar. Remember to rename the `.sh` script also. Naming this just `makedb` can be quite ambiguous.

Reply:
I've changed it to `makedb-walt`